src/store/modules/job/actions.ts

```diff
@@ -751,6 +751,35 @@ const actions: ActionTree<JobState, RootState> = {
     } catch (err) {
       console.error(err);
     }
+  },
+  async fetchMoreJobs({ commit }, payload) {
+    const params = {
+      "inputFields": {
+        ...payload.inputFields
+      } as any,
+      "fieldList": [ "systemJobEnumId", "runTime", "tempExprId", "parentJobId", "serviceName", "jobId", "jobName", "currentRetryCount", "statusId", "productStoreId", "runtimeDataId", "enumName", "shopId", "description", "enumTypeId" ],
+      "noConditionFind": "Y",
+      "viewSize": process.env.VUE_APP_VIEW_SIZE,
+      "viewIndex": 0,
+    }
+
+    try {
+      const resp = await JobService.fetchJobInformation(params)
+      if (resp.status === 200 && !hasError(resp) && resp.data.docs?.length > 0) {
+        const moreJobs = resp.data.docs.map((job: any) => {
+          return {
+            ...job,
+            'status': job?.statusId
+          }
+        })
+        commit(types.JOB_MORE_UPDATED, moreJobs);
```

Comment:
Make the payload consistent, either pass the total property in payload or don't pass the total property(https://github.com/hotwax/job-manager/pull/250/files#diff-1edc9153c69edc14cbca5528effde857efdaf6da5ac6048e4d6fc896b11ffcb4R777)

Reply:
Done sir